packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -315,20 +341,9 @@ class SystemRulesTable extends React.Component {
             this.compressRows(rows).sort(
                 (a, b) => {
                     if (direction === SortByDirection.asc) {
-                        if (column === COMPLIANT_COLUMN) {
-                            return a.parent.cells[column].title.props.className === b.parent.cells[column].title.props.className ? 0 :
-                                a.parent.cells[column].title.props.className < b.parent.cells[column].title.props.className ? 1 : -1;
-                        } else {
-                            return a.parent.cells[column].localeCompare(b.parent.cells[column]);
-                        }
-
+                        return String(a.parent.cells[column].original).localeCompare(String(b.parent.cells[column].original));
```

Comment:
Here's an example of code cleaner by using the `original` version of each row. Before we were checking the props of the compliant column